Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -5,36 +5,28 @@
 import StreamChat
 import UIKit
 
-/// Protocol that adds delegate methods specific for `ChatMessageListCollectionView`
-public protocol ChatMessageListCollectionViewDataSource: UICollectionViewDataSource {
-    /// Get date for item at given indexPath
+/// A protocol for `ChatMessageListView` delegate.
+public protocol ChatMessageListViewDataSource: UITableViewDataSource {
+    /// Get date for item at given index path
     /// - Parameters:
-    ///   - collectionView: CollectionView requesting date
-    ///   - indexPath: IndexPath that should be used to get date
-    func collectionView(_ collectionView: UICollectionView, scrollOverlayTextForItemAt indexPath: IndexPath) -> String?
+    ///   - listView: A view requesting date
+    ///   - indexPath: An index path that should be used to get the date
```

Comment:
Requesting data? Instead of date? In both arguments docs

Reply:
the scroll overlay shows the date for the message it intersects with that's from where the naming came